packages/ui-components/src/icons.tsx

```diff
@@ -99,6 +101,14 @@ export const pyIcon = new LabIcon({
   name: 'elyra:pyIcon',
   svgstr: pyIconSvg
 });
+export const viewIcon = new LabIcon({
+  name: 'elyra:view',
+  svgstr: viewIconSvg
+});
+export const viewOffIcon = new LabIcon({
+  name: 'elyra:viewOff',
+  svgstr: viewOffIconSvg
+});
```

Comment:
what lib did we get these icon from? do we need to add a license reference for them?

Reply:
It comes from carbon components (I think that's where we've gotten our other icons so it should be fine?)